src/AaveV3PriceFeedsUpdate_20230626/AaveV3PolPriceFeedsUpdate_20230626_PayloadTest.t.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {AaveV3Polygon, AaveV3PolygonAssets} from 'aave-address-book/AaveV3Polygon.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {TestWithExecutor} from 'aave-helpers/GovHelpers.sol';
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveV3PolPriceFeedsUpdate_20230626_Payload} from './AaveV3PolPriceFeedsUpdate_20230626_Payload.sol';
+
+contract AaveV3PolPriceFeedsUpdate_20230626_PayloadTest is ProtocolV3TestBase, TestWithExecutor {
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('polygon'));
```

Comment:
@rustboyar , plz always add a current block in tests otherwise it's non deterministic